doc/configuration.rst

```diff
@@ -143,6 +143,56 @@ You can specify the following options in module configuration.
         position = 'center'
     }
 
+.. note::
+    New in version 3.16
+
+You can specify the options in module or py3status configuration section.
+
+The following options will work on ``i3``.
+
+``align``: Specify how modules should be aligned when the ``min_width``
+``background``: Specify a background color for modules.
```

Comment:
please rephrase : `Specify a background color for py3status modules.`

setting a background setting on i3status modules will break i3status (as of today)

Reply:
Done. Not mixed up anymore, right?